scriptworker/test/__init__.py

```diff
@@ -15,7 +15,7 @@
 from scriptworker.config import get_unfrozen_copy, apply_product_config
 from scriptworker.constants import DEFAULT_CONFIG
 from scriptworker.context import Context
-from scriptworker.utils import makedirs
+from scriptworker.utils import makedirs, noop_sync
```

Comment:
We're importing here so other test modules can import from here?

Reply:
That's right. Do you think tests should directly import from `scriptworker.utils`?

Reply:
Eh, both work. Flake8 would probably complain if I hadn't turned it off for `scriptworker/test`; since we're using the imported function, this works.